packages/integration-tests/cmr.js

```diff
@@ -262,6 +285,15 @@ async function generateAndStoreCmrUmmJson(granule, collection, bucket, additiona
     }));
   }
 
+  const defaultVersion = 1.4;
+  if (Number(versionString) > defaultVersion) {
+    jsonObject.MetadataSpecification = {
+      URL: `https://cdn.earthdata.nasa.gov/umm/granule/v${versionString}`,
+      Name: 'UMM-G',
+      Version: `${versionString}`
```

Comment:
This can just be `versionString`, doesn't need to be `${versionString}`

Reply:
heh.  of course